pokemongo_bot/__init__.py

```diff
@@ -523,6 +532,7 @@ def _register_events(self):
     def tick(self):
         self.health_record.heartbeat()
         self.cell = self.get_meta_cell()
+        inventory.refresh_inventory()
```

Comment:
Refresh_inventory will trigger packet sent to server which is not the same behavior as original.